package.json

```diff
@@ -21,15 +21,25 @@
     "reflect-metadata": "^0.1.13",
     "ts-trueskill": "^3.3.0",
     "typeorm": "^0.3.7",
-    "typescript": "^4.8.4",
```

Comment:
`typescript` & `ts-node` are normal dependencies as they are used to transpile during deployment instead of transpile first then pushing and deploying, please revert

Reply:
Done

Reply:
on the main branch ts-node is in devDep only. 
REverting to be consistent with main